portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -195,7 +195,7 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 			<liferay-ui:message arguments="<%= new Object[] {numberFormat.format(start + 1), numberFormat.format(end), numberFormat.format(total)} %>" key="showing-x-to-x-of-x-entries" />
 		</p>
 
-		<nav aria-label="pagination">
+		<nav aria-label="<liferay-ui:message key="pagination" />">
```

Comment:
I'm a bit surprised that this works. I would expect a scriplet here, something like
```jsp
aria-label="<%= LanguageUtil.get(request, "pagination") %>"
```
But if it works, :+1: